linked portraits to players.vue

gerberbaby opened the pull request:
positioned player names on portrait
placed button (non-functional) on portrait
changed opacity of bottom links in host and join.vue

gerberbaby committed:
linked portraits to players.vue
positioned player names on portrait
placed button (non-functional) on portrait
changed opacity of bottom links in host and join.vue

peterchang04 reviewed (commented):
Nice work. A few changes for Players.vue
Start by adding position: relative; to .player, then position: absolute; to each of the child elements of .player.
What this does is tell all .player's children to respect .player as a parent container when absolutely positioning themselves.

The children of .player have been defined using vw thus far, but since the .player div (parent) has been defined by vw already, it may be appropriate the child elements to use % instead. This way, we can tweak the dimensions of .player using vw, and expect all the child elements will scale according their parent, .player.

Talk to me if you need guidance on this.

Look at the individual line comments i've made as well. Do the coding on 7/31 branch still, when finished simply push and we can use this same pull request still

peterchang04 commented on src/components/Players.vue:
```diff
@@ -1,33 +1,74 @@
+    position: relative;
+    width: 6vw;
+    height: 6vw;
+    bottom: 41vw;
```
use top: instead of bottom: it's more intuitive since closer to top than bottom.

peterchang04 commented on src/components/Players.vue:
```diff
@@ -1,33 +1,74 @@
+    font-size: 5.5vw;
+    bottom: 7vw;
+  }
+  #editButton {
```
change this to #cornerButton
We'll want the functionality to sometimes be remove (if you are the host) and sometimes edit (if you are that player)

peterchang04 reviewed (changes requested):
Please make some updates

gerberbaby committed:
changed .player position to relative
changed .player children position to absolute
restyled .player children with % instead of vw
changed use of top/bottom/left/right based on intuitiveness
renamed editButton to cornerButton
in README moved install node.js and install yarn above build project 
using yarn

gerberbaby committed:
changed playerName width to 100%, removed margins, and added padding.

peterchang04 commented on src/components/Players.vue:
```diff
@@ -17,7 +17,7 @@
     <div class="player">
       <portrait4 class="portrait" />
-      <div id="playerName">player 4</div>
+      <div id="playerName">player 4sdfassadfasd</div>
```
change this back i htink. Probably testing for long strings
